include/api/api.h

```diff
@@ -185,6 +185,6 @@ int lf_tag_compare(tag_t tag1, tag_t tag2);
 /**
  * Return the current tag, a logical time, microstep pair.
```

Comment:
This documentation needs enhancement. What does the argument mean? What does a NULL argument mean?

Reply:
DOne